CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.4.0]
+### Added
+- Allow to retain tags history in case of renamed packages ([#157](https://github.com/MetaMask/auto-changelog/pull/157))
```

Comment:
Just a proposal for the commit
- Retain tag history for renamed packages with new validation options ([#157](https://github.com/MetaMask/auto-changelog/pull/157))

  - Introduced --version-before-package-rename and --tag-prefix-before-package-rename options for validate command.